cmd/cmd.go

```diff
@@ -10,56 +10,62 @@ import (
 )
 
 const (
-	EnvLaunchDir     = "PACK_LAUNCH_DIR"
-	EnvBuildpacksDir = "PACK_BUILDPACKS_DIR"
-
-	EnvOrderPath = "PACK_ORDER_PATH"
-	EnvGroupPath = "PACK_GROUP_PATH"
-	EnvPlanPath  = "PACK_PLAN_PATH"
-
-	EnvRunImage = "PACK_RUN_IMAGE"
-
-	EnvUseDaemon  = "PACK_USE_DAEMON"
-	EnvUseHelpers = "PACK_USE_HELPERS"
+	DefaultLaunchDir            = "/workspace"
+	DefaultCacheDir             = "/cache"
+	DefaultBuildpacksDir        = "/buildpacks"
+	DefaultPlatformDir          = "/platform"
+	DefaultOrderPath            = "/buildpacks/order.toml"
+	DefaultGroupPath            = `./group.toml`
+	DefaultPlanPath             = "./plan.toml"
+	DefaultUseDaemon            = false
+	DefaultUseCredentialHelpers = false
```

Comment:
How about `DefaultUseCredHelpers` to match the convention elsewhere?

Reply:
makes sense, I just changed it

Reply:
👍